<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>901</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Agregar</string>
     </attribute>
     <widget class="QPushButton" name="agregar">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>260</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nombre_Line">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="precio_Line">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>70</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>80</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Precio:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>160</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Descripcion:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="descripcion_Line">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>150</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>150</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Descuento:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dato_Line">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>150</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>70</y>
        <width>141</width>
        <height>28</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Descuento</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cantidad</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>662</x>
        <y>70</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Eliminar</string>
     </attribute>
     <widget class="QPushButton" name="eliminar">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>150</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nombre_Line_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Mostrar</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>60</y>
        <width>331</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="agregar_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Mostrar Informacion</string>
      </property>
     </widget>
     <widget class="QPushButton" name="agregar_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>190</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingreso Mostrar</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>925</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
